bno055.h

```diff
@@ -16,6 +16,7 @@
 
 #define BNO055_I2C_ADDR_HI 0x29
 #define BNO055_I2C_ADDR_LO 0x28
+#define BNO055_I2C_ADDR    BNO055_I2C_ADDR_LO

```

Comment:
We need to remove address code from that pull request. Thats different feature

Reply:
Sorry I miss that line. I'll fix it